package excepciones;

public class Ejercicio3 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			lanzarError();
		} catch (Excepcion1 e) {
			e.printStackTrace();
		}
		
	}

	private static void lanzarError() throws Excepcion1{
		// TODO Auto-generated method stub
		
	}

}
